wiki/en/en-Getting-Started.md

```diff
@@ -37,11 +37,9 @@ Now that you have the basics down, let's go:
 
 Also - [Jamulus OS!](https://sourceforge.net/projects/jamulus-os/files/JamulusOS/) Run Jamulus on any computer using a USB stick (64-bit systems only). Checks your hardware and sets you up like a boss straight out of the box!
 
-## All installed?
```

Comment:
We don't need to link the hardware setup yet since Jamulus is probably not yet installed.

Reply:
Agreed.